### ISSUE TITLE
Use MLBlocks for methods backend

### ISSUE THREAD

**Issue by micahjsmith** [HDI-Project/ATM#113]
Currently, ATM defines learning methods (classification only) in json files within the repo. The MLBlocks/MLPrimitives projects provide similar functionality, but include many more methods, including neural networks, gradient boosted machines, and regression models (#72). We should explore using these projects as the backend for learning methods.

**Comment by micahjsmith** [comment]
Here are some thoughts on implementation:
- For each method in `atm/methods/*.json`, find a corresponding primitive in `HDI-Project/MLPrimitives/mlblocks_primitives`, such as RandomForestClassifier https://github.com/HDI-Project/MLPrimitives/blob/master/mlblocks_primitives/sklearn.ensemble.RandomForestClassifier.json
- Replace `atm/methods/` dir with `atm.constants.primitives` or similar, a list of strings identifying MLPrimitives
- Use MLBlocks to make a pipeline: https://github.com/HDI-Project/ATM/blob/master/atm/model.py#L95

**Comment by kard1980** [comment]
Hello I see everyone talking about ATM I have the solution come to Morocco they are very open to hacking simple with a small object more advantageous on carbana Hacked Debit Card?
